release/_package.sh

```diff
@@ -132,6 +132,23 @@ function package_release {
 
 	lc_cd "${_BUILD_DIR}/release"
 
+	if [[ "${_PRODUCT_VERSION}" == 7.3* ]]
```

Comment:
@pgydev, can you please fix the spelling of the word **dependencies** in your commit message?

Thanks